entity-framework/core/miscellaneous/collations-and-case-sensitivity.md

```diff
@@ -14,13 +14,13 @@ Text processing in databases can be complex, and requires more user attention th
 
 ## Introduction to collations
 
-A fundamental concept in text processing is the *collation*, which is a set of rules determining how text values are ordered and compared for equality. For example, while a case-insensitive collation disregards differences between upper- and lower-case letters for the purposes of equality comparison, a case-sensitive collation does not. However, since case-sensitivity is culture-sensitive (e.g. `i` and `I` represent different letters in Turkish), there exist multiple case-insensitive collations, each with its own set of rules. The scope of collations also extends beyond case-sensitivity, to other aspects of character data; in German, for example, it is sometimes (but not always) desirable to treat `ä` and `ae` as identical. Finally, collations also define how text values are *ordered*: while German places `ä` after `a`, Swedish places it at the end of the alphabet.
+A fundamental concept in text processing is the *collation*, which is a set of rules determining how text values are ordered and compared for equality. For example, while a case-insensitive collation disregards differences between upper- and lower-case letters for the purposes of equality comparison, a case-sensitive collation does not. However, since case-sensitivity is culture-sensitive (e.g. `i` and `I` represent different letters in Turkish), multiple case-insensitive collations exist, each with there own set of rules. The scope of collations also extends beyond case-sensitivity, to other aspects of character data; in German, for example, it is sometimes (but not always) desirable to treat `ä` and `ae` as identical. Finally, collations also define how text values are *ordered*: while German places `ä` after `a`, Swedish places it at the end of the alphabet.
```

Comment:
"there" is incorrect - you probably meant "their". I believe the current "its" is better.

The other fix is stylistic ("there exist" is fine), I'd rather we kept it as it is.

Reply:
Thanks. Yes I meant "their". I thought it read easier, but decided to leave it as-is.